diff --git a/js/Plugin/jstree/hm.jstree.js b/js/Plugin/jstree/hm.jstree.js
index 9a82c5a..4e8c861 100644
--- a/js/Plugin/jstree/hm.jstree.js
+++ b/js/Plugin/jstree/hm.jstree.js
@@ -22,8 +22,10 @@
           let reload = true;
           let rollback = false;
           let themes = {
-              dots: false,
-              name: 'default'
+            name: 'default',
+            // variant: 'large',
+            dots: false,
+            icons: true
           };
           let options;
           var offset = 0;
diff --git a/src/Plugin/HmDisplayPlugin/HmDisplayJstree.php b/src/Plugin/HmDisplayPlugin/HmDisplayJstree.php
index e91b0e0..790742e 100644
--- a/src/Plugin/HmDisplayPlugin/HmDisplayJstree.php
+++ b/src/Plugin/HmDisplayPlugin/HmDisplayJstree.php
@@ -42,10 +42,9 @@

       // Search input.
       $form['search'] = [
-        '#type' => 'textfield',
+        '#type' => 'search',
         '#title' => $this
         ->t('Search'),
-        '#description' => $this->t('Type in the search keyword here to filter the tree below. Multiple keywords separated by spaces. Empty the keyword to reset the tree.'),
         '#attributes' => [
           'name' => 'jstree-search',
           'id' => isset($parent_id) ? 'hm-jstree-search-' . $parent_id : 'hm-jstree-search',
@@ -61,11 +60,12 @@

       $form['jstree'] = [
         '#type' => 'html_tag',
-        '#suffix' => '<div class="description">' . $this->t('Click an item to edit it. Drag and drop items to change their position in the tree.') . '</div>',
+        '#prefix' => '<div class="description form-item__description">' . $this->t('Click an item to edit it. Drag and drop items to change their position in the tree.') . '</div>',
         '#tag' => 'div',
         '#value' => '',
         '#attributes' => [
           'class' => [
+            'form-item',
             'hm-jstree',
           ],
           'id' => isset($parent_id) ? 'hm-jstree-' . $parent_id : 'hm-jstree',
